rero_mef/marctojson/do_gnd_agent.py

```diff
@@ -31,6 +31,15 @@
     'u': 'Unknown'
 }
 
+RECORD_TYPES = {
+    'p': 'bf:Person',
+    'b': 'bf:Organisation',
```

Comment:
Is it normal that the both lines are identical?

Reply:
'b' and 'f' are organisations.